digdag-ui/console.jsx

```diff
@@ -355,6 +355,13 @@ function attemptCanResume (attempt) {
   return attempt.done && !attempt.success
 }
 
+function attemptCanKill (attempt) {
+  if (!attempt) {
+    return false
+  }
+  return !attempt.done && !attempt.success
```

Comment:
An attempt can be killed only when attempt is not done. `&& !attempt.success` is unnecessary. Instead, we need to check `attempt.cancelRequested` flag because having `cancelRequested` flag means that it's already kill-requested.

Reply:
I have understood the indicated matter described in your comment.  
Thank you very much for the review and your follow-up.